pkg/cmd/server/kubernetes/master.go

```diff
@@ -63,8 +63,9 @@ func (c *MasterConfig) InstallAPI(container *restful.Container) []string {
 
 // RunNamespaceController starts the Kubernetes Namespace Manager
 func (c *MasterConfig) RunNamespaceController() {
-	// TODO: Add OR c.ControllerManager.EnableDeploymentController once we have upstream deployments
-	experimentalMode := c.ControllerManager.EnableExperimental
+	// we now have several of the kube "experimental" pieces enabled in Origin, so this needs to be
+	// enabled whenever we have the "experimental" APIs enabled.
+	experimentalMode := c.Master.EnableExp
```

Comment:
> Why not set c.ControllerManager.EnableExperimental in the same spot we do c.Master.EnableExp?

To be clear, I'm not saying this check is wrong, I think its fine.  I'd just like `c.ControllerManager.EnableExperimental` to match.